Parser/Device/AbstractDeviceParser.php

```diff
@@ -741,7 +759,9 @@ abstract class AbstractDeviceParser extends AbstractParser
         'SQ' => 'Santin',
         'SY' => 'Sanyo',
         'S9' => 'Savio',
+        'Y4' => 'SCBC',
```

Comment:
I checked everything, but this brand is not located at all

Reply:
@sgiehl can we do a merge with the master without confirming the brand `SCBC` ?
@liviuconcioiu do you have any information about this brand? (any information, screenshot device or page)

Reply:
@sanchezzzhak I don't have any information about it.